runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -697,6 +697,12 @@ class INTERPRETER_CLASS
 	{
 		void* const jitReturnAddress = VM_JITInterface::peekJITReturnAddress(_currentThread, _sp);
 		UDATA jitVTableOffset = VM_JITInterface::jitVTableIndex(jitReturnAddress, interfaceVTableIndex);
+#if defined(J9VM_OPT_VALHALLA_NESTMATES)
+		if (J9_ARE_ANY_BITS_SET(jitVTableIndex, J9_VTABLE_INDEX_DIRECT_METHOD_FLAG)) {
```

Comment:
jitVTableIndex -> jitVTableOffset (and below).

Reply:
fixed, thanks